### PR TITLE
feat(severity): Run checks by severity

### PR DESCRIPTION
**Pull request by jfagoagas**
### Description

We need to run checks using the severity from the metadata file, using `--severity`

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler [hunk 1]
```diff
@@ -33,14 +34,28 @@ if __name__ == "__main__":
     group.add_argument("-C", "--checks-file", nargs="?", help="List of checks")
     group.add_argument("-s", "--services", nargs="+", help="List of services")
     group.add_argument("-g", "--groups", nargs="+", help="List of groups")
-    group.add_argument("-L", "--list-groups", action="store_true", help="List groups")
+    group.add_argument(
+        "--severity",
+        nargs="+",
+        help="List of severities [informational, low, medium, high, critical]",
```
Comment (MrCloudSec) [inline prowler:40]:
```suggestion
        help="List of severities [informational, low, medium, high, critical]",
        choices=["informational","low","medium","high","critical"]
```

**Review by MrCloudSec — CHANGES_REQUESTED** [review]
Good job! How about forcing the user to put the available severities?

**Comment by jfagoagas** [comment]
> Good job! How about forcing the user to put the available severities?

Totally agree, thanks!

File: prowler [hunk 2]
```diff
@@ -33,14 +34,29 @@ if __name__ == "__main__":
...
+    )
+    list_group.add_argument(
+        "-l", "--list-checks", action="store_true", help="List checks"
+    )
+    list_group.add_argument(
+        "--list-services", action="store_true", help="List services"
+    )
 
     parser.add_argument(
         "-b", "--no-banner", action="store_false", help="Hide Prowler Banner"
```
Comment (jfagoagas) [inline prowler]:
```suggestion
        "-b", "--no-banner", action="store_false", help="Hide Prowler banner"
```

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**